docs/modules/develop/pages/guide_accessibility.adoc

```diff
@@ -0,0 +1,236 @@
+
+For more information, refer to:
+https://www.w3.org/TR/WCAG21/#on-input
+
+==== Utilize ARIA attributes where possible
+
+Many elements that provide interactive functionality on the website require ARIA attributes on them to make them accessible. A couple of example of such cases include:
+
+- An element that shows or hides another element on the page, such as an "accordion" element.
+- Dynamic search forms that update another section on the same page without a complete refresh of the page. The button that initiates the search should indicate which element it is connected with on the same page and once the search completes, the number of results found should be announced for the screen readers.
+- Pagination elements where you have "next" and "previous" links as well as "current" page which is shown right now.
+- Progress bars which have "minimum" and "maximum" values as well as a "current" value.
+- Custom dropdown elements that open a specific section on the page containing the dropdown items and also have currently selected items included in them.
+- Form elements that may have errors on them where the errorneous elements should be indicated for people using assistive technologies.
+- Significant areas on the page should be marked with the correct landmark roles, such as navigations and enclosed regions on the page. Many times you can also find the correct HTML element that indicates its correct role by familiarizing yourself with the landmark roles.
+
+To learn more refer to:
+https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes
+
+==== Elements hidden from the accessibility API
+
+People using accessible technologies do not always need to "see" (or hear) all of the items that users without any disabilities see on the page. A common example of such element is a visual decorative icon within links or buttons that is only used to make some elements more recognizable for people with no visual impairments. People using assistive technologies may have visual impairments or may use the website completely using a screen reader when these visual cues may just confuse those users rather than provide any actual improvement on their user experience.
+
+To hide an element from assistive technologies, use the `aria-hidden="true"` attribute on it. Do not abuse this method to hide almost everything from the website to make it "accessible". The website is not really accessible if everything on it is hidden. Always think whether the element you are hiding provides actually useful functionality for the users. If it does so, you should not hide it. But if the element is only used to provide small decorative enhancements on the user experience, it can be sometimes hidden, such as the example case with the icons within buttons.
```

Comment:
```suggestion
To hide an element from assistive technologies, use the `aria-hidden="true"` attribute on it. Do not abuse this method to hide almost everything from the website to make it "accessible". The website is not truly accessible if everything on it is hidden. Always think about whether the element you are hiding provides useful functionality for the users. If it does so, you should not hide it. But if the element is only used to provide small decorative enhancements to the user experience, it can be sometimes hidden, such as in the example case with the icons within buttons.
```